frontend/lib/more/more_page.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'menu_item.dart';
+
+class MorePage extends StatelessWidget {
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        padding: EdgeInsets.symmetric(vertical: 24, horizontal: 18),
+        child: Expanded(
+          child: GridView.count(
+              crossAxisCount: 2,
+              shrinkWrap: true,
+              children: [
+                Center(
+                  child: MenuItem(
+                    title: 'About',
+                    icon: Icons.info_outline,
+                    callback: () {
+                      showAboutDialog(
+                          context: context,
+                          applicationIcon: FlutterLogo(),
+                          applicationName: 'Ehrenamtskarte',
+                          applicationVersion: '1.0.0',
```

Comment:
We should consider using [package_info](https://pub.dev/packages/package_info) for that, so that we can not get out of sync with the "real" app version. It is also a "Flutter favorite" package, so probably a good choice.